setup.py

```diff
@@ -59,10 +59,6 @@ def read_file(filename):
       author_email='alexis@notmyidea.org',
       url='https://github.com/spiral-project/ihatemoney',
       packages=find_packages(),
-      package_data={'': ['*.rst', '*.py', '*.yaml', '*.po', '*.mo', '*.html',
```

Comment:
I'm trying to understand the difference between the entries in ``MANIFEST.in`` and the ``package_data`` entries, it's a real mess but using only the manifest file seems to be the recommended way to go (apparently ``package_data`` only works for binary distribution and not source distribution).